gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -264,24 +267,41 @@ def run(argv=None):
     variants = [variants | 'FlattenPartitions' >> beam.Flatten()]
     num_partitions = 1
 
-  for i in range(num_partitions):
-    table_suffix = ''
-    if partitioner and partitioner.get_partition_name(i):
-      table_suffix = '_' + partitioner.get_partition_name(i)
-    table_name = known_args.output_table + table_suffix
-    _ = (variants[i] | 'VariantToBigQuery' + table_suffix >>
-         variant_to_bigquery.VariantToBigQuery(
-             table_name,
-             header_fields,
-             variant_merger,
-             processed_variant_factory,
-             append=known_args.append,
-             update_schema_on_append=known_args.update_schema_on_append,
-             allow_incompatible_records=known_args.allow_incompatible_records,
-             omit_empty_sample_calls=known_args.omit_empty_sample_calls,
-             num_bigquery_write_shards=known_args.num_bigquery_write_shards,
-             null_numeric_value_replacement=(
-                 known_args.null_numeric_value_replacement)))
+  if known_args.output_table:
+    for i in range(num_partitions):
+      table_suffix = ''
+      if partitioner and partitioner.get_partition_name(i):
+        table_suffix = '_' + partitioner.get_partition_name(i)
+      table_name = known_args.output_table + table_suffix
+      _ = (variants[i] | 'VariantToBigQuery' + table_suffix >>
+           variant_to_bigquery.VariantToBigQuery(
+               table_name,
+               header_fields,
+               variant_merger,
+               processed_variant_factory,
+               append=known_args.append,
+               update_schema_on_append=known_args.update_schema_on_append,
+               allow_incompatible_records=known_args.allow_incompatible_records,
+               omit_empty_sample_calls=known_args.omit_empty_sample_calls,
+               num_bigquery_write_shards=known_args.num_bigquery_write_shards,
+               null_numeric_value_replacement=(
+                   known_args.null_numeric_value_replacement)))
+
+  if known_args.output_avro_path:
+    # TODO(bashir2): Add an integration test that outputs to Avro files and
+    # also imports to BigQuery. Then import those Avro outputs using the bq
+    # tool and verify that the two tables are identical.
+    _ = variants | 'FlattenPartitions' >> beam.Flatten() \
```

Comment:
Seems that I forgot to address the second comment of yours here in my previous pass: Yes `Flatten` may have happened before but IIUC `variants` is still a list of PCollections, although it may only have one element. Or to put it differently, do you see any problems with this code right now?

Reply:
If both are called, it may have an error (something like Transform does not have a stable unique label).